package.json

```diff
@@ -1,22 +1,21 @@
 {
   "dependencies": {
+    "@scalar-labs/scalardl-javascript-sdk-base": "^1.0.0",
     "google-protobuf": "^3.6.1",
-    "grpc-web": "^1.0.4",
-    "jsrsasign": "^8.0.12"
+    "grpc-web": "^1.0.4"
   },
   "name": "@scalar-labs/scalardl-web-client-sdk",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
It's currently 1.0.0, so 1.0.1 maybe ?

Reply:
Sorry, didn't notice that the package version is 1.0.0.
Updated !